main.go

```diff
@@ -145,8 +148,9 @@ func main() {
 
 	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.AccessKey.ID, "aws.accesskey.id", "", "ID of the AWS access key")
 	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.AccessKey.Secret, "aws.accesskey.secret", "", "Secret of the AWS access key")
-	// TODO(nhlfr): Deprecate this option when cert-operator will be implemented.
-	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.CertsDir, "aws.certsdir", "", "Certificated to be placed in /etc/kubernetes/ssl")
+	// TODO(nhlfr): Deprecate these options when cert-operator will be implemented.
+	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.CertsDir, "aws.certsdir", "", "Certificates to be placed in /etc/kubernetes/ssl")
+	daemonCommand.PersistentFlags().StringVar(&Flags.Aws.PubKeyFile, "aws.pubkeyfile", path.Join(os.Getenv("HOME"), ".ssh", "id_rsa.pub"), "Public key to be imported as a keypair in AWS")
```

Comment:
This link describes how to create users manually and put the stuff into authorized_keys by hand, it isn't about using AWS API for that (which is impossible, since `ReserveInstanceInput` has a `KeyName` field with `string` type).